cfgov/jinja2/v1/_includes/atoms/checkbox-list.html

```diff
@@ -0,0 +1,52 @@
+{# ==========================================================================
+
+   common_form_fields.checkbox_list()
+
+   ==========================================================================
+
+   Description: Returns a date picker.
```

Comment:
Fixed in https://github.com/cfpb/cfgov-refresh/commit/65fd027c19ba7c5096d15a32912caee61807f7c4